src/tools/pub/dependencies.md

```diff
@@ -442,6 +439,32 @@ To identify dependencies in your app or package that
 aren't on the latest stable versions,
 use [`dart pub outdated`][].
 
+### Use tight dev_dependencies
```

Comment:
We tend to use "dependency" to refer to the package depended on, so it's not really the dependencies themselves that are "loose" or "tight", but rather the version constraint we specify for each of them.

Perhaps: 

```suggestion
### Use tight version constraints for dev dependencies
```

Reply:
```suggestion
### Tighten version constraints for dev dependencies
```